.github/workflows/trigger-qa-build.yml

```diff
@@ -6,16 +6,78 @@ on:
       workflow:
         description: 'Desired Workflow'
         required: true
-        default: 'LernaPublish'
+        default: 'Lerna Publish'
         type: choice
         options:
-        - LernaPublish
-        - Lerna Publish All Forced
+        - Lerna Publish
+        - Lerna Force Publish Components
+        - Lerna Force Publish Design
+        - Lerna Force Publish Components & Design
+        - Lerna Force Publish All
```

Comment:
Needs hooks somewhere in here as well. 

Side note: I think I get why some packages get a version bump without any changes. It's because if (for example) hooks get a version bump, components would also have a version bump to update its package.json's @jobber/hook version.

With that said, if you have a force design and force hook option, it'll automatically force bump components as well